joblib/test/test_numpy_pickle.py

```diff
@@ -628,6 +629,40 @@ def test_joblib_compression_formats():
                     os.remove(dump_filename)
 
 
+def test_zlibfile_compression_validity():
```

Comment:
I would rename this to `test_external_decompressor_interop` to be more explicit.